<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadadeAken</class>
 <widget class="QDialog" name="RadadeAken">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sisestage radade numbrid</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="rajaSilt">
     <property name="text">
      <string>Rada</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="nimeSilt">
     <property name="text">
      <string>Nimi</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="rada1Kast">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="rada1Nimi">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="rada2Kast">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="rada2Nimi">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="rada3Kast">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="rada3Nimi">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLineEdit" name="rada4Kast">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="rada4Nimi">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLineEdit" name="rada5Kast">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="rada5Nimi">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLineEdit" name="rada6Kast">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="rada6Nimi">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLineEdit" name="rada7Kast">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="rada7Nimi">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLineEdit" name="rada8Kast">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="rada8Nimi">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>335</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RadadeAken</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RadadeAken</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
